### DISCUSSION TITLE
Mitgabe einer Angabe beim ServiceRequest zu welchem besuch die Anforderung gehört

### DISCUSSION THREAD

**Discussion by bNaumann-gem** [gematik/spec-E-Rezept-ServiceRequest#68] [Q&A] [answered]
Bei ambulanten Fällen kann der Patient mehrfach im Quartal zu einem Fall untersucht werden. Wenn erst zum Ende des Quartals die ServiceRequests alle gesammelt (zu einem Fall) geschickt werden, ist es schwer diese ggf. im KIS zum richtigen Besuch zuzuweisen. 

Ggf. könnte hier ein weiterer optionaler Identifier helfen?!

Klärung in eRP-in-KH-Sprechstunde am 30.11.2023

**Comment by florianschoffke** [comment]
Es können weitere Identifier an den ServicerRequest angehangen werden.
s. https://simplifier.net/guide/erp-servicerequest-implementation-guide/Home/Identifier/Identifier-Festlegungen.page.md?version=current

**Comment by bNaumann-gem** [comment]
Kenny Schüller (Cerner) macht einen Vorschlag, wie ein neuer Identifier aussehen kann.

**Comment by florianschoffke** [comment]
@bNaumann-gem 
Kann diese Frage geschlossen werden?

**Reply by bNaumann-gem** [reply]
Nein, Ich würde auf den Vorschlag von Kenny Schüller warten wollen.

**Comment by bNaumann-gem** [comment, marked as answer]
Wie beschrieben, können weitere Identifier genutzt werden:
- Rezept-ID
- Fall-ID
- Vorgangs-ID
Zusätzlich können weitere freiwählbare Identifier ebenfalls im https://simplifier.net/erezept-servicerequest/erp-service-request-prescription-request angegeben werden.
